test/integration/sqlalchemy/test_suite.py

```diff
@@ -2,18 +2,132 @@
 from inspect import cleandoc
 
 import pytest
-from sqlalchemy import create_engine
+from sqlalchemy import (
+    create_engine,
+    testing,
+)
+from sqlalchemy.testing.suite import ComponentReflectionTest as _ComponentReflectionTest
 from sqlalchemy.testing.suite import CompoundSelectTest as _CompoundSelectTest
+from sqlalchemy.testing.suite import DifficultParametersTest as _DifficultParametersTest
 from sqlalchemy.testing.suite import ExceptionTest as _ExceptionTest
 from sqlalchemy.testing.suite import ExpandingBoundInTest as _ExpandingBoundInTest
+from sqlalchemy.testing.suite import HasIndexTest as _HasIndexTest
 from sqlalchemy.testing.suite import NumericTest as _NumericTest
 from sqlalchemy.testing.suite import QuotedNameArgumentTest as _QuotedNameArgumentTest
+from sqlalchemy.testing.suite import RowCountTest as _RowCountTest
 from sqlalchemy.testing.suite import *  # noqa: F403, F401
 from sqlalchemy.testing.suite.test_ddl import (
     LongNameBlowoutTest as _LongNameBlowoutTest,
 )
 
 
+class RowCountTest(_RowCountTest):
+    PYODBC_RATIONAL = cleandoc(
+        """
+        pyodbc does not support returning the actual affected rows when executemany is used,
+        the cursor result always will be set to the rowcount = -1 in this case.
+        This also is a valid behaviour according to the python DBAPI specification.
+        For more details see also:
+        * https://peps.python.org/pep-0249/ 
+        * https://peps.python.org/pep-0249/#rowcount
+        * https://peps.python.org/pep-0249/#id21
+        * https://peps.python.org/pep-0249/#executemany
+        """
+    )
+
+    TURBODBC_RATIONAL = cleandoc(
```

Comment:
```suggestion
    TURBODBC_RATIONALE = cleandoc(
```
Also change references accordingly.

Reply:
fixed